docs/extensions/worker.md

```diff
@@ -278,7 +278,7 @@ Our cluster-autoscaler only needs to know the minimum and maximum number of repl
 Gardener deploys this autoscaler if there is at least one worker pool that specifies `max>min`.
 In order to know how it needs to configure it, the provider-specific `Worker` extension controller must expose which `MachineDeployment`s it has created and how the `min`/`max` numbers should look like.
 
-Consequently, your controller should write this information into the `Worker` resource's `.status.machineDeployments` field:
+Consequently, your controller should write this information into the `Worker` resource's `.status.machineDeployments` field. It should also update the `.status.machineDeploymentsLastUpdateTime` field along with `.status.machineDeployments`, so that gardener is able to deploy cluster-autoscaler right after the Worker status is updated with the latest machine deployments and does not wait for the Worker reconciliation to be completed:
```

Comment:
```suggestion
Consequently, your controller should write this information into the `Worker` resource's `.status.machineDeployments` field. It should also update the `.status.machineDeploymentsLastUpdateTime` field along with `.status.machineDeployments`, so that gardener is able to deploy Cluster-Autoscaler right after the status is updated with the latest `MachineDeployment`s and does not wait for the reconciliation to be completed:
```